app/views/modals/set-home-page-modal.html

```diff
@@ -0,0 +1,50 @@
+<div class="modal-resource-action">
+  <div class="modal-header">
+    <h1>Set Home Page</h1>
+  </div>
+  <div class="modal-body modal-body-indent">
+    <p>
+      <div class="title">Set your preferred home page.</div>
+      Note: This setting is browser specific and will not be maintained across browsers.
+    </p>
+    <ng-form name="homePageForm">
+      <fieldset ng-disabled="loading">
+        <div class="radio">
+          <label>
+            <input type="radio" ng-model="preferredHomePage" value="catalog-home">
+            Catalog Home (Default)
+          </label>
+        </div>
+        <div class="radio">
+          <label>
+            <input type="radio" ng-model="preferredHomePage" value="project-list">
+            My Projects List
+          </label>
+        </div>
+        <div class="radio" ng-show="availableProjects.length >= 1">
+          <label>
+            <input type="radio" ng-model="preferredHomePage" value="project-overview">
+            Overview Page for {{availableProjects.length === 1 ? (selectedProject | displayName) : ':'}}
+          </label>
+          <div class="select-project-container">
+            <select-project ng-if="availableProjects.length > 1"
+                            class="select-project"
```

Comment:
Does this class have any styles? I don't see it in this PR.

Reply:
Removed, not needed